### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 21

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#347] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325512-dfc3a6d0-3174-4738-b255-400c9092a6fd.png)

**Comment by Exyss** [comment]
![Esercizio 21](https://user-images.githubusercontent.com/65043207/152679125-385e2cbd-1a58-439e-9b35-2f41f8385c36.jpg)

**Comment by aflaag** [comment]
è corretta
